.github/workflows/best-practice.yml

```diff
@@ -20,9 +20,8 @@ jobs:
       uses: actions/setup-java@v1
       with:
         java-version: 1.8
-    - name: Run tests
+    - name: Run best practice tests
       env:
         SAUCE_USERNAME: ${{ secrets.SAUCE_USERNAME }}
         SAUCE_ACCESS_KEY: ${{ secrets.SAUCE_ACCESS_KEY }}
-        SCREENER_API_KEY: ${{ secrets.SCREENER_API_KEY }}
-      run: mvn test -pl best-practice -Dtest=Visual*
+      run: mvn test -pl best-practice -X
```

Comment:
@titusfortner I don't think we want to run the entire best-practice suite. With the instability of Sauce in Desktop and especially Mobile, it's likely to lead to false positive failures and slow CI feedback

Reply:
> instability of Sauce in Desktop
Tell me more about what you're seeing.

If there's stuff we don't want run, we should tag the specific tests so it is obvious.

Reply:
@titusfortner I think that visual is failing because you removed `SCREENER_API_KEY` from the .yml

Reply:
That's just, like, your opinion, man.